documentation/boltspec_reference.md

```diff
@@ -4,6 +4,180 @@ The `BoltSpec` helper library includes several functions to help write unit
 tests for plans. For more information about writing unit tests for plans, and
 how to use these functions, see [Testing plans](testing_plans.md).
 
+## Any stubs
+
+Plans often execute sub-plans with the `run_plan` function to build complex
+workflows. When testing these plans, it might be helpful to execute any
+sub-plans as well without needing to stub or mock the plan. To support this,
+`BoltSpec` offers two different modes:
+
+### `allow_any_plan`
+
+The `allow_any_plan` function stubs the [`run_plan`
+function](plan_functions.md#run-plan). It does not accept any parameters.
+
+The `allow_plan` function accepts the following modifiers:
+
+- `be_called_times(number)`
+
+  The test fails if any plan is run more than _number_ times.
```

Comment:
Is it that each individual plan can be called up to _number_ times, or that you can only have up to _number_ `run_plan` invocations?

Reply:
I went [here](https://github.com/puppetlabs/bolt/blob/main/spec/bolt_spec/plan_spec.rb#L404) and changed `allow_any_plan` to `allow_any_plan.be_called_times(1)` and the test passed. Since this plan calls another plan, I think its safe to say that each plan can be called up to _number_. Thanks for the clarifying question😌